### PR TITLE
ansible: update to 2.10.1.

### PR DESCRIPTION
**Pull request by jcgruenhage**
Ansible has been split into ansible-base and ansible, where ansible just contains modules and depends on ansible-base. There's still some warnings (both during build and install) I'd like to have someone with more python knowledge look over, but I've already deployed this on my workstation, works just fine.

### CODE REVIEW

File: srcpkgs/ansible/template [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
+version=2.10.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="${hostmakedepends} python3-cryptography python3-Jinja2 python3-paramiko
- python3-yaml"
+depends="python3 ansible-base"
 checkdepends="python3-pytest"
 short_desc="Simple deployment, configuration management and execution framework"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
```
Comment (jcgruenhage) [inline srcpkgs/ansible/template:11]: Talked with @the-maldridge on IRC, they're in favour of me taking over.

File: srcpkgs/ansible-base/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'ansible-base'
+pkgname=ansible-base
+version=2.10.2
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="${hostmakedepends} python3-cryptography python3-Jinja2 python3-paramiko
+ python3-yaml"
+checkdepends="python3-pytest"
+short_desc="Simple deployment, configuration management and execution framework"
```
Comment (jcgruenhage) [inline srcpkgs/ansible-base/template:10]: Not sure, does it make sense to add a different description here, and if so, which one?

File: srcpkgs/ansible/template (hunk 1 above)
Comment (the-maldridge) [reply srcpkgs/ansible/template:11]: :+1:

File: srcpkgs/ansible/template [hunk 3]
```diff
@@ -1,23 +1,14 @@
...
-checksum=3ab21588992fbfe9de3173aefd63da1267dc12892a60f5cfdc055fe19c549644
-
-post_install() {
-	vsconf examples/ansible.cfg
-	vsconf examples/hosts
-	for m in docs/man/man1/*.1; do
-		vman ${m}
-	done
-}
+distfiles="https://pypi.python.org/packages/source/a/ansible/ansible-${version}.tar.gz"
```
Comment (PaperMountainStudio) [inline srcpkgs/ansible/template:13]: maybe `${PYPI_SITE}/a/ansible/ansible-${version}.tar.gz`?

Comment (jcgruenhage) [reply srcpkgs/ansible/template]: done

**Pull request closed without merging**